src/groupREADME.md

```diff
@@ -0,0 +1,76 @@
+##Abstract:
+Many people have routine eating pallets and  what's cooking good looking offers many unique recipes as well as cooking instructions to advance a persons culinary experience. 
+
+What's Cooking Good Looking: is a web application that provides a user the abilities of:  being able to search a recipe, save a recipe, delete a saved recipe,filter all recipes,filter all saved recipes, search by tags, and see specifications on any app compiled recipe. 
+
+##Installation Instructions:
+1. start at https://github.com/daniabee/whats-cookin-good-lookin
+2. click CODE
+3. Copy SSH 
+4. Go to computer terminal type: git clone <'filename'> 
+5. cd into directory 
+6. run 'npm install'
+7. run 'npm start'
+8. should be given a server page 'http://localhost:8080/'
+9. open server in browser.  
+
+
+##Preview of App:
+
+##Context:
+Each member of our group has been in Turing for 3 months || 13 weeks. 
+
+##Where you can find the spec:
+
+https://frontend.turing.edu/projects/whats-cookin-part-one.html
+
+##Contributors:
+Andrew Cerveny : https://github.com/AndrewCerveny
+Brett Kuhn: https://github.com/bkuhn2
+Courtney Lippman : https://github.com/Courtney-Lippman
+Dani Bagley: https://github.com/daniabee
+
+
+
+
+##Learning Goals:
+1. To work in a group atmosphere 
+2. Experience Api fetching 
+3. Using array Iterators in functions to get desired behavior
+4. Creating a wire frame
+5. Use TDD testing to build data model
+6. Learn the abilities of WebPack 
+7. Practice Import and exports
+8. Practice Git Work FLow 
+9. Practice creating a uniform Project Management system 
+
+##Technology Used
+VS Code
+JavaScript
+Mocha
+Chai
+Github
+Terminal
+WebPack
+HTML
+CSS
+
+
+##Wins:
+A consistent win would be the limitation of merge conflicts. Each member created their own test, their own class, and then their own HTML Page. We would organize our work flow through daily standup's and as we merged we  minimized the possibilities of major merge conflicts. Our second major win was with the retrieval of our api data and allowing it to be accessible outside of the .then promise data trap. 
+
+##Challenges:
+The greatest challenges were...
+
+1. The ability to access api data after the promise was resolved. 
+2. The ability for functions to be on page load and how that interacts with Api retrieval. 
+3. Group scheduling across time zones.
+4. Making sure commits seemed balanced. 
+Future Features:
```

Comment:
Maybe make Future Features a heading as well (##).

Reply:
Nice spot another oversight I made the corrections!